client/src/components/Leaderboard.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import MaterialTable from './MaterialTable';
+
+const styles = theme => ({
+  root: {
+    width: '50%',
+    minWidth: 200,
+    margin: `${theme.spacing.unit * 2}px auto`,
+    padding: theme.spacing.unit * 2,
+  },
+  title: {
+    fontSize: theme.typography.h5.fontSize,
+  },
+});
+
+class Leaderboard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      leaderboards: [],
+    };
+  }
+
+  componentDidMount() {
+    const { getTopFive } = this.props;
+    getTopFive()
+      .then(({ data: leaderboards }) => {
+        this.setState({
+          leaderboards,
+        });
+      });
+  }
+
+  render() {
+    const { classes } = this.props;
+    const { leaderboards } = this.state;
+    if (leaderboards.length) {
+      const leaderboardsRows = leaderboards.map(leader => (
+        <TableRow key={leader.username}>
+          <TableCell>{leader.username}</TableCell>
```

Comment:
Also, this could be made into separate component call `TableRow` or something so less clutter here.

Reply:
I did not want to make 2 components doing the same work. `TableRow` already exists in material-ui.

Reply:
okay.